typescript/src/purchasesConfiguration.ts

```diff
@@ -93,9 +93,8 @@ export interface PurchasesConfiguration {
 
   /**
    * Override the preferred UI locale for RevenueCat UI components at runtime. This affects both API requests and UI rendering.
-   * If the locale changes, this will automatically clear the offerings cache and trigger a background refetch to get paywall templates with the correct localizations.
```

Comment:
Hmm maybe keep it but with something like:
```
This will automatically clear the offerings cache and trigger a background refetch to get paywall templates with the correct localizations.
```